tests/test_classic_entropy_ranking.cpp

```
@@ -0,0 +1,240 @@
+    resid* mock_residue = nullptr;
+    gridpoint* mock_cleftgrid = nullptr;
+    BindingPopulation* pop = nullptr;
+    static constexpr double T = 300.0;
+    static constexpr double EPS = 1e-6;
+    static constexpr int N_CHROM = 40;
+
+    void SetUp() override {
+        mock_fa = new FA_Global();
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wnontrivial-memcall"
+        std::memset(mock_fa, 0, sizeof(FA_Global));
```

Cppcheck (2.21.0)

[error] Using 'memset' on struct that contains a 'std

(memsetClass)